src/server/routes/pathways/generate-pathway-json/biopax-metadata/index.js

```diff
@@ -35,18 +36,30 @@ let extractBiopaxMetadata = biopaxJsonEntry => {
     type: type.includes(':') ? type.split(':')[1] : type,
     standardName,
     displayName,
-    databaseIds
+    xrefLinks
   };
 };
 
+// helper for converting xrefs to map of uris grouped by db
+const xref2link = ( db, id, xrefLinks ) => {
+  return xref2Uri( db, id )
+    .then( uri => {
+      if( xrefLinks[ db ] != null ){
+        xrefLinks[ db ] = xrefLinks[ db ].concat( uri );
```

Comment:
This relates to #704. I am just sticking all these xrefs in here and letting the client decide. Right now the client just picks the first one for each db...

Reply:
I am concerned that this would mean that we would be making potentially hundreds of requests to generate this pathway json now.

Reply:
Fair enough! Alternatives:
  - use the external-service-data mapping within `xref2Uri` rather than live service
  - get @IgorRodchenkov to spin up an endpoint to take in multi-pairs of xrefs

Reply:
I'd prefer the second option.  The convertor is nice for consistency but it needs to be fast.